defi/jest.config.js

```diff
@@ -19,11 +22,4 @@ module.exports = {
   ci: false,
   cacheDirectory: '/tmp/jest-cache',
   testEnvironment: "node",
```

Comment:
hmm, unsure if this will break current github action flow that has a dependency on test passing before deploying to aws